scripts/serve.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/zsh
+
+WD=$(greadlink -f $(dirname "$0"))
```

Comment:
We should use the standard `readlink`, no? Unrecognized `greadlink` was the reason I couldn't run your code the other day.